tests/helper.py

```diff
@@ -136,6 +136,10 @@ def check_all_connected_from_inputs(onnx_model):
     # Nodes which are not connected from the network inputs.
     orphan_nodes = []
     for node in onnx_model.graph.node:
+        if node.op_type == 'Constant':
```

Comment:
I think a slightly better solution is `not node.input`, since ONNX may add another constant op again. ONNX used to have `ConstantLike` and `ConstantFill`, which may not have any inputs.

Reply:
good point, thx